src/bin/vip-cache-purge-url.js

```diff
@@ -6,14 +6,20 @@ import * as exit from '../lib/cli/exit';
 import { readFromFile } from '../lib/read-file';
 import { trackEvent } from '../lib/tracker';
 
+const usage = 'vip cache purge-url';
+const exampleUsage = 'vip @example-app.develop cache purge-url';
+
 const examples = [
 	{
-		usage: 'vip cache purge-url <URL>',
-		description: 'Purge a URL from page cache',
+		usage:
+			`${ exampleUsage } https://example-app-develop.go-vip.co/sample-page/` +
+			'\n    - Purged URL: https://example-app.develop.go-vip.co/sample-page/',
+		description: 'Purge the page cache for a single URL.',
```

Comment:
![Screenshot](https://github.com/Automattic/vip-cli/assets/139488/f1d425fe-560b-4aeb-890f-ed2e0ee999f9)

Reply:
I just tested this and can confirm that the example output matches the actual output when that command is run successfully. 👍